pilot/cmd/pilot-agent/status/util/http.go

```diff
@@ -24,7 +24,8 @@ import (
 
 const requestTimeout = time.Second * 2
 
-func doHTTPGet(requestURL string) (*bytes.Buffer, error) {
+// DoHTTPGet performs an HTTP GET to the requested URL and returns the response as a buffer.
```

Comment:
s/an/a/

Reply:
It is actually `an`. HTTP starts with a vowel sound.